README.md

```diff
@@ -1 +1,31 @@
-# Awesome-books-with-ES6
\ No newline at end of file
+### Awesome-books-with-ES6 and modules:
+
+- Recreation of the Awesome Books Project using modules and ES6 syntax.
+
+![imagen](https://user-images.githubusercontent.com/93448210/148102835-146307af-594e-42df-909c-22a943232d2a.png)
+
+# Build with
+
+- HTML
+- CSS
+- Javascript
+ 
+![imagen](https://user-images.githubusercontent.com/93448210/148102496-ed20c529-bf6b-4eee-8b5c-1cf9f1cb4bdd.png)
+
+
```

Comment:
- [ ] I would recommend adding the steps to clone and run the project locally in the README file to make the README professional.

Hint:
```
git clone https://github.com/AlejandroNeira98/Awesome-books-with-ES6.git

cd Awesome-books-with-ES6

open folder in vs code and open `index.html` with live server.
```